### PR TITLE
`Popup:` Fixes TextPopupElement composable shifting in size when coming back into visibility

### PR DESCRIPTION
**Pull request by eri9000**
### Description:
- When a TextPopupElement is scrolled out of view and back into view, its width is recalculated multiple times and it can cause it to push other elements out of view.

### CODE REVIEW

File: toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/internal/element/textelement/TextPopupElement.kt [hunk 1]
```diff
@@ -81,7 +81,9 @@ private fun HTML(content: String) {
             val completeHtml = "$header$headStyle<html>${content.trim()}</html>"
             loadDataWithBaseURL(null, completeHtml, "text/html", "UTF-8", null)
         }
-    })
+    },
```
Comment (puneet-pdx) [inline toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/internal/element/textelement/TextPopupElement.kt:84]: Worth adding a comment here to explain the empty implementation. Something on the lines which states, 
```
By default, AndroidView does not automatically pool or reuse Views. If placed inside of a reusable container (including inside a LazyRow or LazyColumn), the View instances will always be discarded and recreated if the composition hierarchy containing the AndroidView changes, even if its group structure did not change and the View could have conceivably been reused.
If onReset is null or not specified, the AndroidView instance will not support reuse, and the View instance will always be discarded whenever the AndroidView is moved or removed from the composition hierarchy.
```

File: toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/Popup.kt [hunk 2]
```diff
@@ -150,35 +150,42 @@ private fun PopupBody(popupState: PopupState, onFileClicked: (ViewableFile?) ->
...
+                    item(contentType = FieldsPopupElement::class.java) {
                         FieldsPopupElement(
                             entry.state as FieldsElementState,
                         )
                     }
+                }
 
-                    is MediaPopupElement -> {
+                is MediaPopupElement -> {
+                    item(contentType = MediaPopupElement::class.java) {
```
Comment (puneet-pdx) [inline toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/Popup.kt:180]: Add comment for contentType addition? 
```
contentType is needed for reuse of AndroidView inside a LazyColumn
```

**Review by puneet-pdx — APPROVED** [review]
@eri9000 Looks good. Just some suggestions to add doc for these additions, won't hold the PR for those.

**Review by shubham7109 — APPROVED** [review]
Looks good 👍

**Pull request merged**